lib/taskana-core-test/src/test/java/acceptance/classification/query/ClassificationQueryPaginationAccTest.java

```diff
@@ -0,0 +1,139 @@
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").list(0, -1);
+    assertThat(results).hasSize(0);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetZeroClassifications_When_OffsetAndLimitAreZero() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").list(0, 0);
+    assertThat(results).hasSize(0);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetOneClassifications_When_OffsetIsNegativeAndLimitIsPositive() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").list(-1, 1);
+    assertThat(results).hasSize(1);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetFirstFiveClassificationsOnAPage_When_FirstPageAndPageSizeIsFive() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").listPage(1, 5);
+    assertThat(results).hasSize(5);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetLastFourClassificationsOnAPage_When_FourthPageAndPageSizeIsFive() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").listPage(4, 5);
+    assertThat(results).hasSize(4);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetAllClassificationsOnAPage_When_FirstPageAndPageSizeIsTwenty() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").listPage(1, 20);
+    assertThat(results).hasSize(19);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetZeroClassificationsOnAPage_When_PageSizeIsZero() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").listPage(1, 0);
+    assertThat(results).hasSize(0);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetZeroClassificationsOnAPage_When_PageSizeIsNegative() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").listPage(1, -1);
+    assertThat(results).hasSize(0);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetClassificationsOnAPage_When_PagenumberIsZeroButPageSizeIsPositive() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").listPage(0, 1);
+    assertThat(results).hasSize(1);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_GetClassificationsOnAPage_When_PagenumberIsNegativeButPageSizeIsPositive() {
+    List<ClassificationSummary> results =
+        classificationService.createClassificationQuery().domainIn("DOMAIN_A").listPage(-1, 1);
```

Comment:
What is the difference to "should_GetOneClassifications_When_OffsetIsNegativeAndLimitIsPositive" ?